integration/localnet/bootstrap.go

```diff
@@ -310,7 +310,7 @@ func prepareServiceDirs(role string, nodeId string) (string, string) {
 	if nodeId != "" {
 		profilerDir = "./" + filepath.Join(ProfilerDir, role, nodeId)
 	}
-	err = os.MkdirAll(profilerDir, 0755)
+	err = os.MkdirAll(profilerDir, 0777)
```

Comment:
can you add a comment on why we're giving full access to all users?

Reply:
This was an attempt at resolving this issue, reverted now that we have a more valid solution.